docs/data-sources/mks_cluster.md

```diff
@@ -78,6 +78,7 @@ Read-Only:
 - `high_availability` (Boolean) Select this option for highly available control plane. Minimum three control plane nodes are required
 - `kubernetes_upgrade` (Attributes) (see [below for nested schema](#nestedatt--spec--config--kubernetes_upgrade))
 - `kubernetes_version` (String) Kubernetes version of the Control Plane
+- `installerTtl` (Integer) By default, this setting allows ttl configuration for installer config. If not provided by default will set ttl to 365 days.
```

Comment:
@venkatm-rafay it should be `installer_ttl`. TF schema follows underscore convention